docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -22,16 +22,28 @@ General-purpose aggregation functions
 ``ArrayAgg``
 ------------
 
-.. class:: ArrayAgg(expression, distinct=False, **extra)
+.. class:: ArrayAgg(expression, distinct=False, ordering=None, **extra)
 
     Returns a list of values, including nulls, concatenated into an array.
+    Ordering within the array is unspecified by default. Has one optional
```

Comment:
"one" seems incorrect. It's best to avoid specifying a number there so that it doesn't have to be updated as things change.